bears/java/CheckstyleBear.py

```diff
@@ -64,6 +72,9 @@ def create_arguments(
             - geosoft - The Java style followed by GeoSoft. More info at
               <http://geosoft.no/development/javastyle.html>
         """
+        # Check for invalid configurations based on selected checkstyle_configs
+        invalid_configuration(checkstyle_configs, use_spaces, indent_size)
```

Comment:
I think I would prepend to the name of the function `check_`, so
--> `check_invalid_configuration`, then you don't need the comment, because the function name is self-descriptive then :)